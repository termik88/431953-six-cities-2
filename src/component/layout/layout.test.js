import React from 'react';
import renderer from 'react-test-renderer';

import Layout from './layout';
jest.mock(`../header/header.jsx`);

describe(`Layout component test.`, () => {
  it(`Correctly renders after relaunch`, () => {
    const mockComponent = <div></div>;
    const mainPage = renderer
      .create(
          <Layout layoutClassName="page">
            {mockComponent}
          </Layout>
      ).toJSON();

    expect(mainPage).toMatchSnapshot();
  });
});
