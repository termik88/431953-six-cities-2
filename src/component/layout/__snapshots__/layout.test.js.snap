// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component test. Correctly renders after relaunch 1`] = `
<div
  className="page page"
>
  <div />
</div>
`;
